crates/trigger/src/runtime_config/key_value.rs

```diff
@@ -95,14 +108,20 @@ impl TriggerHooks for KeyValuePersistenceMessageHook {
         }) {
             return Ok(());
         }
-        // Only print if the 'default' store has type 'spin'.
-        if let KeyValueStoreOpts::Spin(store_opts) = runtime_config.default_key_value_opts() {
-            if let Some(path) = &store_opts.path {
-                println!("Storing default key-value data to {path:?}");
-            } else {
-                println!("Using in-memory default key-value store; data will not be saved!");
+
+        match runtime_config.default_key_value_opts() {
+            KeyValueStoreOpts::Redis(RedisKeyValueStoreOpts { url }) => {
+                println!("Storing default key-value data to Redis at {url:?}");
```

Comment:
Will the url here contain any passwords/secrets?

Reply:
I don't think we necessarily need to print here at all. I wanted to for the defaults because it may not be obvious what's happening but for this you have to explicitly configure it anyway.